src/components/VContentSwitcher/VSearchTypePopover.vue

```diff
@@ -2,7 +2,7 @@
   <VPopover
     ref="contentMenuPopover"
     class="flex items-stretch"
-    :label="$t('search-type.label')"
+    :label="$t('search-type.label').toString()"
```

Comment:
why does localized text need to be converted to string in templates? 🤔

Reply:
Because for TypeScript, i18n does not return string, it returns `TranslationResult`, and it's getting flagged in the templates. It should be fixed in newer versions of i18n library, and hopefully we'll remove all `toString()`s when we update.